helm/leanix-k8s-connector/values.yaml

```
@@ -39,6 +39,10 @@ args:
     container: ""
   blacklistNamespaces:
   - "kube-system"
+  additionalEnv:
+    FOO: "BAR"
+    HTTP_PROXY: "CHANGEME"
+    HTTPS_PROXY: "CHANGEME"
```

The default value should not contain any key value pairs. I will add some examples to the README.md after the merge, when we release the new Helm chart package.

```suggestion
  additionalEnv: {}
```